model/base/src/main/java/org/eclipse/ditto/model/base/exceptions/MissingPayloadException.java

```diff
@@ -0,0 +1,122 @@
+ /*
+  * Copyright (c) 2020 Contributors to the Eclipse Foundation
+  *
+  * See the NOTICE file(s) distributed with this work for additional
+  * information regarding copyright ownership.
+  *
+  * This program and the accompanying materials are made available under the
+  * terms of the Eclipse Public License 2.0 which is available at
+  * http://www.eclipse.org/legal/epl-2.0
+  *
+  * SPDX-License-Identifier: EPL-2.0
+  */
+ package org.eclipse.ditto.model.base.exceptions;
+
+ import java.net.URI;
+
+ import javax.annotation.Nullable;
+ import javax.annotation.concurrent.Immutable;
+ import javax.annotation.concurrent.NotThreadSafe;
+
+ import org.eclipse.ditto.json.JsonObject;
+ import org.eclipse.ditto.model.base.common.HttpStatusCode;
+ import org.eclipse.ditto.model.base.headers.DittoHeaders;
+ import org.eclipse.ditto.model.base.json.JsonParsableException;
+
+ /**
+  * Thrown if a request with missing payload is being made.
+  *
+  * @since 1.5.0
+  */
+ @Immutable
+ @JsonParsableException(errorCode = MissingPayloadException.ERROR_CODE)
+ public final class MissingPayloadException extends DittoRuntimeException {
```

Comment:
I suggest to also rename this exception with the "CloudEvent" prefix to: `CloudEventMissingPayloadException`